lua/plugins/lualine.lua

```diff
@@ -0,0 +1,9 @@
+return {
+ 'nvim-lualine/lualine.nvim',
+ dependencies={{ 'nvim-lualine/lualine.nvim'}},
```

Comment:
The dependency at line 3 can be removed too but it was a bug with the previous version of the config so that's okay.